### ISSUE TITLE
Update dedupe to 2.0.13 on the ogr/develop branch

### ISSUE THREAD

**Issue by jwalgran** [open-apparel-registry/open-apparel-registry#1955]
- Split the dedupe version upgrade out of #1309 so it can be merged into the ogr/develop branch and unblock Python upgrade work
- Rebase the branch for #1309 

--Update 20220721, cs
The work for this issue was completed, though it was not successful. We separated out the version upgrade and merged a PR into staging, at which point we realized there were some performance regressions in the dedupe 2.x branch, which caused the workers to time out before they could train a model and start up cleanly.

The plan now is to hold off on the dedupe version upgrade until we can apply it in tandem with building a model detached from the worker initialization procedure, so training speed will not affect worker availability.
